### PR TITLE
[CDLS] Add max query, fix serialisation issue, add default user

### PR DESCRIPTION
**Pull request by klindziukp**
## PROBLEM
- Need to add max items by default
- Issue when retrieving tests via `http://localhost:8080/api/tests?maxResults=4`
```
"timestamp": 1707144807244,
    "status": 500,
    "error": "Internal Server Error",
    "message": "com.fasterxml.jackson.databind.exc.InvalidDefinitionException: No serializer found for class software.amazon.awssdk.services.dynamodb.model.AttributeValue and no properties discovered to create BeanSerializer (to avoid exception, disable SerializationFeature.FAIL_ON_EMPTY_BEANS) (through reference chain: java.util.Collections$UnmodifiableMap[\"created\"])",
    "path": "/api/tests"
```
- Need to fix TTL creation for local instantiation
## SOLUTION
- Implement util for verification maximum request size
- Disable `SerializationFeature.FAIL_ON_EMPTY_BEANS`
- Enable ttl param for test table
- Add creation default user for test purposes

### CODE REVIEW

File: localstack/create-tables-dynamodb.sh [hunk 1]
```diff
@@ -230,6 +230,5 @@ EOF
 )
 
 echo $(aws --endpoint-url ${ENDPOINT} dynamodb create-table --cli-input-json "$TABLE_SCHEDULE")
-
-
+echo $(aws --endpoint-url ${ENDPOINT} dynamodb update-time-to-live --table-name "$TABLE_SCHEDULE" --time-to-live-specification Enabled=true,AttributeName=e_at --region us-east-1)
```
Comment (Strunevskiy) [inline localstack/create-tables-dynamodb.sh:233]: I am not sure why it is needed.

File: src/main/java/com/codeless/api/automation/util/ValidationUtil.java [hunk 2]
```diff
@@ -0,0 +1,12 @@
+package com.codeless.api.automation.util;
+
+public final class ValidationUtil {
+
+  private ValidationUtil() {
+    throw new RuntimeException();
+  }
+
+  public static int validateMaxResults(int maxResults) {
+    return Math.min(maxResults, 50);
```
Comment (Strunevskiy) [inline src/main/java/com/codeless/api/automation/util/ValidationUtil.java:10]: I would say, getMaxResultsOrDefault.
50 is a lot, let's do it to default, which is 25.

File: src/main/resources/application.properties [hunk 3]
```diff
@@ -1,4 +1,5 @@
 server.servlet.context-path=/api
+spring.profiles.active=local
```
Comment (Strunevskiy) [inline src/main/resources/application.properties:2]: why not set profile in IDE ?

File: src/main/java/com/codeless/api/automation/converter/NextTokenConverter.java [hunk 4]
```diff
@@ -34,7 +35,7 @@ public String toString(Map<String, AttributeValue> nextToken) {
     try {
       return encode(objectMapper.writeValueAsString(nextToken));
     } catch (JsonProcessingException e) {
-      throw new RuntimeException();
```
Comment (Strunevskiy) [inline src/main/java/com/codeless/api/automation/converter/NextTokenConverter.java:37]: very good point.

File: src/main/java/com/codeless/api/automation/util/ValidationUtil.java (hunk 2 above)
Comment (Strunevskiy) [reply src/main/java/com/codeless/api/automation/util/ValidationUtil.java:10]: Also, call this class something else, for me, validation either throws error or return true or false.

**Review by Strunevskiy — APPROVED** [review]

File: src/main/java/com/codeless/api/automation/converter/NextTokenConverter.java [hunk 5]
```diff
@@ -3,6 +3,7 @@
 import com.fasterxml.jackson.core.JsonProcessingException;
 import com.fasterxml.jackson.core.type.TypeReference;
 import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.SerializationFeature;
```
Comment (Strunevskiy) [inline src/main/java/com/codeless/api/automation/converter/NextTokenConverter.java:6]: why do you need this import?

**Review by Strunevskiy — APPROVED** [review]

File: src/main/java/com/codeless/api/automation/converter/NextTokenConverter.java (hunk 5 above)
Comment (klindziukp) [reply src/main/java/com/codeless/api/automation/converter/NextTokenConverter.java:6]: to disable `SerializationFeature.FAIL_ON_EMPTY_BEANS`
        
        ```
return encode(objectMapper.disable(SerializationFeature.FAIL_ON_EMPTY_BEANS)
          .writeValueAsString(nextToken));
```

File: src/main/resources/application.properties (hunk 3 above)
Comment (klindziukp) [reply src/main/resources/application.properties:2]: Good, point. will be removed

File: src/main/java/com/codeless/api/automation/util/ValidationUtil.java (hunk 2 above)
Comment (klindziukp) [reply src/main/java/com/codeless/api/automation/util/ValidationUtil.java:10]: renamed to `DefaultValueUtil.getMaxResultsOrDefault(25)`

File: localstack/create-tables-dynamodb.sh (hunk 1 above)
Comment (klindziukp) [reply localstack/create-tables-dynamodb.sh:233]: to create TTL, because I get errors when trying to create tables

**Pull request merged**